packages/core/src/Stores/ui-store.js

```diff
@@ -89,7 +89,7 @@ export default class UIStore extends BaseStore {
     has_real_account_signup_ended = false;
 
     // verification modal
-    is_verification_modal_visible = false;
+    is_verification_modal_visible = true;
```

Comment:
why do we set default value as true?

Reply:
oops was testing the modal 😓 let me change this back 🙏